command/harness/helper.go

```diff
@@ -35,3 +39,43 @@ func fileID(filename string) string {
 	h.Write([]byte(filename))
 	return strings.Replace(filepath.Base(filename), ".", "-", -1) + strconv.Itoa(int(h.Sum32()))
 }
+
+// validateStruct checks if all fields of a struct are populated.
+func validateStruct(data interface{}) error {
```

Comment:
Is there an alternative to using reflection here?

Reply:
Alternative is to check each field manually